src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.h

```diff
@@ -285,6 +286,7 @@ class QMCFixedSampleLinearOptimizeBatched : public QMCDriverNew, LinearMethod
   // Need to keep this around, unfortunately, since QMCCostFunctionBatched uses QMCCostFunctionBase,
   // which still takes an MCWalkerConfiguration in the constructor.
   MCWalkerConfiguration& W;
+  std::optional<EstimatorManagerInput> global_emi_;
```

Comment:
this needs to be const ref as well. Could you add documentation why this is needed? "This reference is used to initialize VMC driver"

Reply:
I don't want to get in the business of lifetime dependencies with the inputs.  We need make no lifetime guarantees about the input objects when constructing objects anywhere else I want to continue with that.